android-client-sdk/src/main/java/com/devcycle/sdk/android/util/DVCSharedPrefs.kt

```diff
@@ -2,7 +2,7 @@ package com.devcycle.sdk.android.util
 
 import android.content.Context
 import android.content.SharedPreferences
-import com.devcycle.sdk.android.R
+import com.devcycle.R
```

Comment:
I am worried this change will change the location of the prefs and thus the upgrade path is broken for folks without internet, need to test the upgrade path somehow with the same user id

Reply:
Tested the upgrade path by: 
- loading test app with latest released version of the SDK
- reloading app with local version of SDK
- verified that user config is initially loaded from cache